app/src/main/java/com/quran/labs/androidquran/database/BookmarksDBAdapter.java

```diff
@@ -149,6 +149,7 @@ public void addRecentPage(int page) {
   private void addRecentPage(int page, boolean shouldDelete) {
     ContentValues contentValues = new ContentValues();
     contentValues.put(LastPagesTable.PAGE, page);
+    contentValues.put(LastPagesTable.ADDED_DATE, System.currentTimeMillis() / 1000);
```

Comment:
not necessary - it defaults to `TIMESTAMP DEFAULT CURRENT_TIMESTAMP`

Reply:
When i tried to getTimestamp (via getLong) i always get 2019 for some reason, when i print string it show that this is stored as date, and maybe because of that i get only 2019 (as start of string). I couldn't found for sure why this happening only with LastePages :/